demos/select.html

```diff
@@ -179,37 +180,42 @@ <h2 class="mdc-typography--title">CSS Only</h2>
     <script>
       (function() {
         var MDCSelect = mdc.select.MDCSelect;
-        var root = document.getElementById('js-select');
-        var currentlySelected = document.getElementById('currently-selected');
+        var heroSelect = document.getElementById('hero-js-select');
```

Comment:
Even if everything fully supported es2015 syntax overnight, I don't think it would be a very good use of time to re-implement the demos just for the new syntactic sugars available to us. As delicious as they may be. :)